src/server/pfs/server/driver.go

```diff
@@ -158,33 +158,37 @@ func newLocalDriver(blockAddress string, etcdPrefix string) (*driver, error) {
 	return newDriver(blockAddress, []string{"localhost:32379"}, etcdPrefix, defaultTreeCacheSize)
 }
 
-// initializePachConn initializes the connects that the pfs driver has with the
+// getPachClient() initializes the connection that the pfs driver has with the
 // Pachyderm object API and auth API, and blocks until the connection is
 // established
 //
 // TODO(msteffen): client initialization (both etcd and pachd) might be better
 // placed happen in server.go, near main(), so that we only pay the dial cost
 // once, and so that pps doesn't need to have its own initialization code
-func (d *driver) initializePachConn() error {
+func (d *driver) getPachClient(ctx context.Context) *client.APIClient {
 	d.pachConnOnce.Do(func() {
-		d.pachConn, d.onceErr = grpc.Dial(d.address, client.PachDialOptions()...)
+		var err error
+		d.pachConn, err = grpc.Dial(d.address, client.PachDialOptions()...)
 		d.pachClient = &client.APIClient{
 			AuthAPIClient:   auth.NewAPIClient(d.pachConn),
 			ObjectAPIClient: pfs.NewObjectAPIClient(d.pachConn),
 		}
+		if err != nil {
+			panic(fmt.Sprintf("could not intiailize Pachyderm client in driver: %v", err))
+		}
```

Comment:
So what is the desired recourse? If on startup the driver cannot successfully dial we panic and force a restart? Is there a reason this isn't in a backoff (or at least backoff pending the network is retryable pattern)?